src/main/java/com/cburch/logisim/std/io/HexDigitHDLGeneratorFactory.java

```diff
@@ -30,65 +30,67 @@
             .pair("busName", GetBusName(componentInfo, HexDigit.HEX, nets))
             .pair("bubbleBusName", bubbleBusName)
             .pair("startId", startId)
+            .pair("startId6", startId + 6)
             .pair("regName", LineBuffer.format("s_{{1}}_reg", componentInfo.getComponent().getAttributeSet().getValue(StdAttr.LABEL)))
             .pair("sigName", LineBuffer.format("{{1}}[{{2}}:{{3}}]", bubbleBusName, (startId + 6), startId))
             .pair("dpName", GetNetName(componentInfo, HexDigit.DP, true, nets));
-
     if (HDL.isVHDL()) {
       contents.add("");
       if (componentInfo.isEndConnected(HexDigit.HEX)) {
         contents
-            .add("WITH ({{busName}}) SELECT {{bubbleBusName}}( {{1}} DOWNTO {{startId}} ) <= ", (startId + 6))
-            .addLines(
-                "   \"0111111\" WHEN \"0000\",",
-                "   \"0000110\" WHEN \"0001\",",
-                "   \"1011011\" WHEN \"0010\",",
-                "   \"1001111\" WHEN \"0011\",",
-                "   \"1100110\" WHEN \"0100\",",
-                "   \"1101101\" WHEN \"0101\",",
-                "   \"1111101\" WHEN \"0110\",",
-                "   \"0000111\" WHEN \"0111\",",
-                "   \"1111111\" WHEN \"1000\",",
-                "   \"1100111\" WHEN \"1001\",",
-                "   \"1110111\" WHEN \"1010\",",
-                "   \"1111100\" WHEN \"1011\",",
-                "   \"0111001\" WHEN \"1100\",",
-                "   \"1011110\" WHEN \"1101\",",
-                "   \"1111001\" WHEN \"1110\",",
-                "   \"1110001\" WHEN OTHERS;");
+            .add("""
+                 WITH ({{busName}}) SELECT {{bubbleBusName}}({{startId6}} DOWNTO {{startId}}) <= 
+                    0111111" WHEN "0000",
```

Comment:
still the " before the 0111111" is missing, also for the lines below

Reply:
Fixed.